app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorHeader.vue

```diff
@@ -257,6 +282,20 @@ export default {
       //Avoid errors when user goes to the same route
       // eslint-disable-next-line no-unused-vars
       ).catch(err => {})
+    },
+    enabledInputQueries() {
+      const userId = getUserId();
+      if (!this.enabledForkButton && !!userId) {
```

Comment:
Why is the user necessary?

Reply:
It's unnecessary. It's enough to check if the user is logged.